City-of-Helsinki/drupal-asuntotuotanto
# SATT-54: rename alkuperainen asumioikeusmaksu and new field for indeksikorotuksen maara

### Description

- Remove new field Haso Fee
- Rename Indeksikorotuksen määrä to Alkuperäinen asumisoikeusmaksu
- New field for Indeksikorotuksen määrä


#### How to test it

- make fresh 
- login as admin
- check that apartment listing search api indexing is working https://asuntotuotanto.docker.so/fi/admin/config/search/search-api/index/apartment_listing
- check that haso and hitas listing pages are still working
  - https://asuntotuotanto.docker.so/fi/asuntohaku/hitas
  - https://asuntotuotanto.docker.so/fi/asuntohaku/haso
- On Haso listing page and haso project& apartment page
  - Check that asumisoikeusmaksu is showing
  - Also check that the mobile view is working correctly and showing the correct values
